src/SFA.DAS.Employer.PR.Web/Controllers/Requests/DeclineCreateAccountConfirmationController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Employer.PR.Web.Constants;
+using SFA.DAS.Employer.PR.Web.Infrastructure;
+using SFA.DAS.Employer.PR.Web.Infrastructure.Services;
+using SFA.DAS.Employer.PR.Web.Models;
+using SFA.DAS.Employer.PR.Web.Models.Session;
+
+namespace SFA.DAS.Employer.PR.Web.Controllers.Requests;
+
+
+[Route("requests")]
+public class DeclineCreateAccountConfirmationController(ISessionService _sessionService) : Controller
+{
+    [Authorize]
+    [HttpGet]
+    [Route("{requestId:guid}/createaccount/decline/confirmed", Name = RouteNames.DeclineCreateAccountConfirmation)]
+    public IActionResult Index(Guid requestId, CancellationToken cancellationToken)
+    {
+        Request.HttpContext.Items.Add(SessionKeys.AccountTasksKey, true);
+        var sessionModel = _sessionService.Get<AccountCreationSessionModel>();
```

Comment:
Check for session model is null in case of browser back. If it is null we have option to navigate to page not found  or to accounts home or throw exception, I am open to suggestion as I need to manage this on success confirmation as well.

Reply:
Ah yes, good idea.  Will just throw to Check page if it is null